arch/arm/mach-imx/imx_secondary_boot.c

```diff
@@ -48,3 +48,17 @@ U_BOOT_CMD(
 	"   0 - set primary image\n"
 	"   1 - set secondary image\n"
 );
+
+static int do_warm_reset(cmd_tbl_t *cmdtp, int flag,
+			 int argc, char * const argv[])
+{
+	call_imx_sip(IMX_SIP_WARM_RESET, 0, 0, 0, 0);
```

Comment:
why dont we check the return value?

Reply:
well, we can do that, but it doesn't really make sense as technically we will never return from this call (because of reset done in TF-A)

Reply:
so then no return on the function ?